server/controllers/users.js

```diff
@@ -37,7 +48,13 @@ class UserController {
       await user.createProfile();
 
       const token = await Token.create(payload);
-      return res.status(201).json({ status: 'success', message: 'User created successfully', user: userExtractor(user, token) });
+      return res
```

Comment:
you can use the Response class here. Response.success()

Reply:
@EMacco  Any reason why this feedback was ignored?